code/controllers/subsystem/minimaps.dm

```diff
@@ -656,5 +675,22 @@ SUBSYSTEM_DEF(minimaps)
 	marker_flags = MINIMAP_FLAG_MARINE_SOM
 
 /datum/action/minimap/observer
-	minimap_flags = MINIMAP_FLAG_XENO|MINIMAP_FLAG_MARINE|MINIMAP_FLAG_MARINE_SOM|MINIMAP_FLAG_EXCAVATION_ZONE
+	minimap_flags = MINIMAP_FLAG_ALL // What's the point of making `MINIMAP_FLAG_ALL` if `ALL` exists? Unless this runtimes for some reason.
 	marker_flags = NONE
```

Comment:
A) minimap flag all only applies to max current existing flags and not ALL, this is necesseary because we use bitfields2list in minimap init
B) this should be explicit imo because there can easily be cases where we have information bloat later on if people add more flags

Reply:
looking at code rn looks like we use it too much anyway